general/base_import/import_faq.rst

```diff
@@ -0,0 +1,201 @@
+
+
+What can I do if I have multiple matches for a field?
+-----------------------------------------------------
+
+If for example you have two product categories with the child name "Sellable" (ie. "Misc. Products/Sellable" & "Other Products/Sellable"), your validation is halted but you may still import your data. However, we recommend you do not import the data because they will all be linked to the first 'Sellable' category found in the Product Category list ("Misc. Products/Sellable"). We recommend you modify one of the duplicates' values or your product category hierarchy.
+
+However if you do not wish to change your configuration of product categories, we recommend you use make use of the external ID for this field 'Category'.
+
+
+
+How can I import a many2many relationship field (e.g. a customer that has multiple tags)?
+-----------------------------------------------------------------------------------------
+
+The tags should be separated by a comma without any spacing. For example, if you want your customer to be linked to both tags 'Manufacturer' and 'Retailer' then you will encode "Manufacturer,Retailer" in the same column of your CSV file.
+
+`CSV file for Manufacturer, Retailer <../../_static/example_files/m2m_customers_tags.csv>`_.
+
+
+
+How can I import a one2many relationship (e.g. several Order Lines of a Sales Order)?
+-------------------------------------------------------------------------------------
+
+If you want to import sales order having several order lines; for each order line, you need to reserve a specific row in the CSV file. The first order line will be imported on the same row as the information relative to order. Any additional lines will need an addtional row that does not have any information in the fields relative to the order.
+As an example, here is purchase.order_functional_error_line_cant_adpat.CSV file of some quotations you can import, based on demo data.
+
+`File for some Quotations <../../_static/example_files/purchase.order_functional_error_line_cant_adpat.csv>`_.
+
+The following CSV file shows how to import purchase orders with their respective purchase order lines:
+
+`Purchase orders with their respective purchase order lines <../../_static/example_files/o2m_purchase_order_lines.csv>`_.
+
+The following CSV file shows how to import customers and their respective contacts:
+
+`Customers and their respective contacts <../../_static/example_files/o2m_customers_contacts.csv>`_.
+
+
+
+Can I import several times the same record?
+-------------------------------------------
+
+If you import a file that contains one of the column "External ID" or "Database ID", records that have already been imported will be modified instead of being created. This is very usefull as it allows you to import several times the same CSV file while having made some changes in between two imports. Odoo will take care of creating or modifying each record depending if it's new or not.
```

Comment:
I think this is a very important part. I would rephrase it to say this is why you **should** put an external id when creating your CSV.